README.md

```diff
@@ -212,6 +212,41 @@ const App = () => (
 );
 ```
 
+### Access props in actions
```

Comment:
This is great, but what do you think about changing the example just using React and Redux Zero, just like the other examples? Recompose and lodash are not used by everybody, we should keep it as simple as possible

Reply:
I like examples to be explicit, using (solving) real world problems. But I'll change it to your preferences, no problem